app/views/leader_sessions/sign_in.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Sign in</h1>
+
+<form action="/verify_credentials" method="post">
+  <div>
+    <label for="email">Email</label>
+    <input id="email" name="email" placeholder="Enter your email..." type="text">
+  </div>
+
+  <div>
+    <label for="password">Password</label>
+    <input id="password" name="password" placeholder="Choose a password..." type="password">
```

Comment:
The placeholder here needs updating? They've already chosen a password when signing up.